pom.xml

```diff
@@ -658,6 +658,9 @@
           <groupId>org.apache.maven.plugins</groupId>
           <artifactId>maven-jar-plugin</artifactId>
           <configuration>
+            <excludes>
+              <exclude>**/hadoop-metrics2-accumulo.properties</exclude>
+            </excludes>
```

Comment:
This should probably exist only in the test module, rather than in the root pom.

Reply:
Should I add a complete maven-jar-plugin to the test module pom?  I looked there but I did not see one configured. I thought about it, but was unsure if a new plugin configuration or promoting it to the parent would be preferred.  

Having it in the parent does have a side benefit that if any module ends up needing a metrics file for testing it would be excluded by default.

It's your call as to the preferred approach.

Reply:
Placing the plugin section in the test module pom with this configured there would be preferred, so it doesn't unintentionally apply to other modules, and also so it's more visible to people modifying the test module in the future, in case they modify it in a way that is impacted by this configuration.

Other modules wouldn't be affected anyway, since they would use `src/test/resources`. The only reason the test module is affected is because it uses `src/main/resources`. If we actually wanted to intentionally put this file in another module's `src/main/resources`, we wouldn't want this configuration to accidentally omit it.